boards/tests/test_views.py

```diff
@@ -0,0 +1,19 @@
+from django.urls import reverse
+from django.urls import resolve
+from django.test import TestCase
+
+# imports from this folder
+from boards.views import board_list
+
+
+class BoardListViewTests(TestCase):
+    """It tests list of Boards"""
+
+    def test_home_view_produces_status_code_200(self):
+        url = reverse("home")
+        response = self.client.get(url)
```

Comment:
Tried calling view but It was not working so, I had to use client.

Reply:
Now I relaize, I can use RequestFactory to avoid middleware and exception . I am going to change this line by RequestFactory.